lib/encodings.js

```diff
@@ -1,4 +1,6 @@
-var Buffer = require('buffer').Buffer
+'use strict'
+
+const Buffer = require('buffer').Buffer
```

Comment:
```suggestion
const { Buffer } = require('buffer')
```

would be more modern